scripts/misc/generate_reg_file.pl

```diff
@@ -95,6 +96,9 @@ =head1 CONTACT
     funcgen => {
         adaptor => 'Bio::EnsEMBL::Funcgen::DBSQL::DBAdaptor',
     },
+    #otherfeatures => {
+    #    adaptor => 'Bio::EnsEMBL::DBSQL::DBAdaptor',
+    #},
```

Comment:
Just need to uncomment
```suggestion
    otherfeatures => {
        adaptor => 'Bio::EnsEMBL::DBSQL::DBAdaptor',
    },
```

Reply:
It's generating an error for me with comments, i.e:

With comment:
```sh
(base) [diegomscoelho@codon-login-02 1250]$ reg --release 107 --species human --otherfeatures v3
#!/usr/bin/env perl
use Bio::EnsEMBL::Registry;
use Bio::EnsEMBL::DBSQL::DBAdaptor;
use Bio::EnsEMBL::Funcgen::DBSQL::DBAdaptor;
use Bio::EnsEMBL::Variation::DBSQL::DBAdaptor;


  Bio::EnsEMBL::Variation::DBSQL::DBAdaptor->new(
    '-species' => 'Homo_sapiens',
    '-group'   => 'variation',
    '-port'    => 4606,
    '-host'    => 'mysql-ens-var-prod-3',
    '-user'    => 'ensadmin',
    '-pass'    => 'ensembl',
    '-dbname'  => 'homo_sapiens_variation_107_38_tmptables',
  );
Use of uninitialized value in printf at /hps/software/users/ensembl/repositories/diegomscoelho/ensembl-variation/scripts/misc/generate_reg_file.pl line 226.

  ->new(
    '-species' => 'Homo_sapiens',
    '-group'   => 'otherfeatures',
    '-port'    => 4606,
    '-host'    => 'mysql-ens-var-prod-3',
    '-user'    => 'ensro',
    '-pass'    => '',
    '-dbname'  => 'homo_sapiens_otherfeatures_107_38',
  );

Bio::EnsEMBL::Registry->add_alias('Homo_sapiens', 'human'); 

1;
```

Without comment:
```sh
(base) [diegomscoelho@codon-login-02 1250]$ reg --release 107 --species human --otherfeatures v3
#!/usr/bin/env perl
use Bio::EnsEMBL::Registry;
use Bio::EnsEMBL::DBSQL::DBAdaptor;
use Bio::EnsEMBL::Funcgen::DBSQL::DBAdaptor;
use Bio::EnsEMBL::DBSQL::DBAdaptor;
use Bio::EnsEMBL::Variation::DBSQL::DBAdaptor;


  Bio::EnsEMBL::Variation::DBSQL::DBAdaptor->new(
    '-species' => 'Homo_sapiens',
    '-group'   => 'variation',
    '-port'    => 4606,
    '-host'    => 'mysql-ens-var-prod-3',
    '-user'    => 'ensadmin',
    '-pass'    => 'ensembl',
    '-dbname'  => 'homo_sapiens_variation_107_38_tmptables',
  );

  Bio::EnsEMBL::DBSQL::DBAdaptor->new(
    '-species' => 'Homo_sapiens',
    '-group'   => 'otherfeatures',
    '-port'    => 4606,
    '-host'    => 'mysql-ens-var-prod-3',
    '-user'    => 'ensro',
    '-pass'    => '',
    '-dbname'  => 'homo_sapiens_otherfeatures_107_38',
  );

Bio::EnsEMBL::Registry->add_alias('Homo_sapiens', 'human'); 

1;
```